### PR TITLE
OSDOCS-8119: Retaining S3 bucket

### PR DESCRIPTION
**Pull request by nalhadef**
Version(s):
4.14

Issue:
https://issues.redhat.com/browse/OSDOCS-8119

Link to docs preview:
[Direct to blurb](https://66576--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-aws-s3-deletion)
[Top level](https://66576--docspreview.netlify.app/)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -72,12 +72,14 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 [id="ocp-4-14-installation-and-update"]
 === Installation and update
 
+[id="ocp-4-14-AWS-S3-Deletion"]
```
Comment (kalexand-rh) [inline release_notes/ocp-4-14-release-notes.adoc:75]:
```suggestion
[id="ocp-4-14-aws-s3-deletion"]

```

Anchors need to be [lowercase](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata), and you need a blank line after the [heading](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#document-header).

**Review by kalexand-rh — COMMENTED** [review]
@nalhadef, please make these formatting changes and rebase.

**Comment by nalhadef** [comment]
merge-review-needed

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -76,8 +76,14 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 [id="ocp-4-14-aws-shared-vpc"]
 ==== Installing a cluster on Amazon Web Services (AWS) using a shared VPC
+
 In {product-title} 4.14, you can install a cluster on AWS that uses a shared VPC, with a private hosted zone in a different account than the cluster. For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[Installing a cluster on AWS into an existing VPC].
 
+[id="ocp-4-14-aws-s3-deletion"]
+==== Enabling S3 bucket to be retained during cluster bootstrap on AWS
+
+With this release users are able to opt out of the automatic deletion of the S3 bucket during the cluster bootstrap on Amazon Web Services. This is key for customers with security policies preventing the deletion of S3 buckets.
```
Comment (abhatt-rh) [inline release_notes/ocp-4-14-release-notes.adoc:85]: One of the recommendation from the Red Hat Suppl guide for [short description](https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc) suggests:
> Use customer-centric language, such as "You can…​ by…​" or "To…​, configure…​".

```suggestion
With this update, you can opt out of the automatic deletion of the S3 bucket during the cluster bootstrap on AWS. This option is useful when you have security policies that prevent the deletion of S3 buckets.
```

**Review by abhatt-rh — COMMENTED** [review]
@nalhadef I noticed that this PR did not have a `peer-review-done` label. So I went ahead completed the peer review and added the required label. I added a comment for your consideration. 
Removing the `merge-review` related labels for now. Please re-queue when you are ready for merge. Thanks!

**Review by jldohmann — APPROVED** [review]
LGTM

**Comment by jldohmann** [comment]
@nalhadef the `peer-review-done` label should remain on all PRs if applicable, it helps tell us that merged PRs went through the process

**Pull request merged**